R/qenv-get_code.R

```diff
@@ -118,19 +118,16 @@ setMethod("get_code", signature = "qenv", function(object, deparse = TRUE, names
   }
 
   code <- if (!is.null(names)) {
+    # todo:
     get_code_dependency(object@code, names, ...)
   } else {
     object@code
   }
 
   if (deparse) {
-    if (length(code) == 0) {
-      code
-    } else {
-      paste(code, collapse = "\n")
-    }
+    code
```

Comment:
@gogonzo 
We don't have `unlist(code)` in here, so you do not loose attributes of `code` in `[.qenv` when you subset the code.
This means in all places you need to run `unlist(get_code())` to get code represented as a character, not as a list.

Reply:
or we could steer the behavior with a parameter, so it unlist for get_code but does not unlist for `[.`